src/OrchardCore.Modules/OrchardCore.UrlRewriting/Views/UrlRedirectRule.Edit.cshtml

```diff
@@ -6,6 +6,11 @@
     <span class="hint">@T["Rewrite rules use regular expression syntax for pattern matching as defined in ECMA-262."]</span>
 </div>
 
+<div class="mb-3">
```

Comment:
Originally fields where grouped using following logical groups:

Match (Pattern field, Ignore case field)
Conditions (no conditions in curent implmentation)
Action (Rewrite/Redirect parameters)

Reply:
reverted